src/pages/journal/journal.ts

```diff
@@ -52,14 +73,14 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
     public toastController: ToastController,
     private store$: Store<StoreModel>,
     private slotSelector: SlotSelectorProvider,
-    private resolver: ComponentFactoryResolver
+    private resolver: ComponentFactoryResolver,
+    private networkState: NetworkStateProvider
```

Comment:
Can we call this `networkStateProvider` so that we know that it's a provider?

Reply:
No longer use `NetworkStateProvider` in the journal page. 👍